File: 0485 max-consecutive-ones/solution.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0, c = 0;
        for (int i : nums) {
        c = i ==1 ? c + 1 : 0;
        if(c>max) max=c;
        }
        return max;
    }
}

File: 0485 max-consecutive-ones/solution.rs
impl Solution {
    pub fn find_max_consecutive_ones(nums: Vec<i32>) -> i32 {
       let mut c=0;
       let mut m=0;
       for i in 0..nums.len() {
        if nums[i]==1 {
            c+=1;
        } else {
            m=m.max(c);
            c=0;
        }
       }
       m.max(c)
    }
}
